final_prep/README.md

```diff
@@ -44,31 +44,31 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1097" alt="Screen Shot 2022-04-20 at 7 55 55 PM" src="https://user-images.githubusercontent.com/80866068/164814995-1da85819-2a28-47b0-ad7d-8d9e6eb838f2.png">
+- <img width="1140" alt="Screen Shot 2022-04-20 at 7 56 08 PM" src="https://user-images.githubusercontent.com/80866068/164815067-f52448f4-111a-417e-becc-5f2f1babd5e8.png">
```

Comment:
This looks like a realistic Turing week. Glad to see your run/outdoor time on there in the AMs!